### ISSUE TITLE
not a bug but just something to improve

### ISSUE THREAD

**Issue by bloohacked** [bypassiwastaken/skiovox-helper#56]
for people who use the system day/month/year (like me cause I am australian) put in an option in the skivox menu to use that system

**Comment by bypassiwastaken** [comment]
good idea but is this really necessary? if it is you can submit a PR as i dont think i wanna put effort into that

**Comment by 3kh0** [comment]
australian is crazy

**Comment by bloohacked** [comment]
Not really but it might be a good thing to add

**Comment by bloohacked** [comment]
I am working on it

**Comment by bypassiwastaken** [comment]
ok

**Comment by bloohacked** [comment]
I made the pr

**Comment by Honkychunkymonkey** [comment]
already going to be added in next PR

**Comment by Honkychunkymonkey** [comment]
> > already going to be added in next PR
> 
> THAT I MADE
mines gonna be better btw ;)

**Comment by bloohacked** [comment]
> > > already going to be added in next PR
> > 
> > 
> > THAT I MADE
> > mines gonna be better btw ;)

what pr

**Comment by Honkychunkymonkey** [comment]
> > > > already going to be added in next PR
> > > 
> > > 
> > > THAT I MADE
> > > mines gonna be better btw ;)
> 
> what pr

you'll see soon you innocent little child

**Comment by Honkychunkymonkey** [comment]
> > > > already going to be added in next PR
> > > 
> > > 
> > > THAT I MADE
> > > mines gonna be better btw ;)
> 
> you can stop "flexing" your shitty PR

I'm so offended that a child who can only develop in html css and js called my PR shitty wahh wahh (im crying so hard rn) wahh wahh

**Comment by bloohacked** [comment]
> > > > > already going to be added in next PR
> > > > 
> > > > 
> > > > THAT I MADE
> > > > mines gonna be better btw ;)
> > 
> > 
> > you can stop "flexing" your shitty PR
> 
> I'm so offended that a child who can only develop in html css and js called my PR shitty wahh wahh (im crying so hard rn) wahh wahh

Really, you have to stop before I flex my coding skills that are probably 6900000% better than yours

**Comment by bloohacked** [comment]
> > > > > already going to be added in next PR
> > > > 
> > > > 
> > > > THAT I MADE
> > > > mines gonna be better btw ;)
> > 
> > 
> > you can stop "flexing" your shitty PR
> 
> I'm so offended that a child who can only develop in html css and js called my PR shitty wahh wahh (im crying so hard rn) wahh wahh

also I only found web code on your profile

**Comment by bloohacked** [comment]
> calm down laddies I am sure we can resolve this

ok please and I just think I should just block that idiot

**Comment by bloohacked** [comment]
> > > calm down laddies I am sure we can resolve this
> > 
> > 
> > ok please and I just think I should just block that idiot
> 
> yall just need to chill, there is no reason for you all to argue about this

yeah, I'm just gonna chill and close the issue cause I don't want to continue this any longer
